examples/privacytenant/example_test.go

```diff
@@ -52,50 +52,50 @@ func Do(ctx context.Context, client *ent.Client) error {
 	if _, err := client.Tenant.Create().Save(viewOnly); !errors.Is(err, privacy.Deny) {
 		return fmt.Errorf("expect operation to fail, but got %w", err)
 	}
-	// Apply the same operation with "Admin" role.
-	admin := viewer.NewContext(ctx, viewer.UserViewer{Role: viewer.Admin})
-	hub, err := client.Tenant.Create().SetName("GitHub").Save(admin)
+	// Apply the same operation with "Admin" role, expect it to pass.
+	adminCtx := viewer.NewContext(ctx, viewer.UserViewer{Role: viewer.Admin})
+	hub, err := client.Tenant.Create().SetName("GitHub").Save(adminCtx)
 	if err != nil {
 		return fmt.Errorf("expect operation to pass, but got %w", err)
 	}
 	fmt.Println(hub)
-	lab, err := client.Tenant.Create().SetName("GitLab").Save(admin)
+	lab, err := client.Tenant.Create().SetName("GitLab").Save(adminCtx)
 	if err != nil {
 		return fmt.Errorf("expect operation to pass, but got %w", err)
 	}
 	fmt.Println(lab)
 
-	// Create 2 users connected to the 2 tenants we created above (a8m->GitHub, nati->GitLab).
-	a8m := client.User.Create().SetName("a8m").SetTenant(hub).SaveX(admin)
-	nati := client.User.Create().SetName("nati").SetTenant(lab).SetFoods([]string{"Sushi", "Burritos"}).SaveX(admin)
+	// Create 2 users connected to the 2 tenants we created above
+	a8mHub := client.User.Create().SetName("a8mHub").SetTenant(hub).SaveX(adminCtx)
+	natiLab := client.User.Create().SetName("natiLab").SetTenant(lab).SetFoods([]string{"Sushi", "Burritos"}).SaveX(adminCtx)
 
 	hubView := viewer.NewContext(ctx, viewer.UserViewer{T: hub})
 	out := client.User.Query().OnlyX(hubView)
-	// Expect that "GitHub" tenant to read only its users (i.e. a8m).
-	if out.ID != a8m.ID {
+	// Expect that "GitHub" tenant to read only its users (i.e. a8mHub).
+	if out.ID != a8mHub.ID {
 		return fmt.Errorf("expect result for user query, got %v", out)
 	}
 	fmt.Println(out)
 
 	labView := viewer.NewContext(ctx, viewer.UserViewer{T: lab})
 	out = client.User.Query().OnlyX(labView)
-	// Expect that "GitLab" tenant to read only its users (i.e. nati).
-	if out.ID != nati.ID {
+	// Expect that "GitLab" tenant to read only its users (i.e. natiLab).
+	if out.ID != natiLab.ID {
 		return fmt.Errorf("expect result for user query, got %v", out)
 	}
 	fmt.Println(out)
 
 	// Expect operation to fail, because the DenyMismatchedTenants rule makes sure
 	// the group and the users are connected to the same tenant.
-	_, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(nati).Save(admin)
+	_, err = client.Group.Create().SetName("entgo.io").SetTenant(hub).AddUsers(natiLab).Save(adminCtx)
 	if !errors.Is(err, privacy.Deny) {
-		return fmt.Errorf("expect operation to fail, since user (nati) is not connected to the same tenant")
+		return fmt.Errorf("expect operation to fail, since user (natiLab) is not connected to the same tenant")
 	}
```

Comment:
nit: limit error scope

here and a few places below. probably out of scope of this pr but leaving comment for posterity :)